frontend/src/utils/og.ts

```diff
@@ -15,21 +15,21 @@ export const createDetailPageMeta = ({
   const meta = [
     {
       hid: "robots",
-      name: "robots",
+      property: "robots",
```

Comment:
The [robots meta tag](https://developers.google.com/search/docs/crawling-indexing/robots-meta-tag#robotsmeta) must use `name` and `content`. Please revert this change.